-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-38366/30557

ERROR: Build errors found during clang-tidy run.
```
CalibPPS/ESProducers/plugins/TotemDAQMappingESSourceXML.cc:251:27: error: no matching member function for call to 'getParameter' [clang-diagnostic-error]
      sampicSubDetId(conf.getParameter<unsigned int>("sampicSubDetId", 5)),
                          ^
/cvmfs/cms-ib.cern.ch/nweek-02737/el8_amd64_gcc10/cms/cmssw-patch/CMSSW_12_5_X_2022-06-14-1100/src/FWCore/ParameterSet/interface/ParameterSet.h:303:26: note: candidate function template not viable: requires single argument 'name', but 2 arguments were provided
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/CalibPPS/ESProducers/plugins/TotemDAQMappingESSourceXML.cc
+++ b/CalibPPS/ESProducers/plugins/TotemDAQMappingESSourceXML.cc
@@ -248,7 +248,7 @@ const string TotemDAQMappingESSourceXML::tagTotemTimingPlane = "timing_plane";
 TotemDAQMappingESSourceXML::TotemDAQMappingESSourceXML(const edm::ParameterSet &conf)
     : verbosity(conf.getUntrackedParameter<unsigned int>("verbosity", 0)),
       subSystemName(conf.getUntrackedParameter<string>("subSystem")),
-      sampicSubDetId(conf.getUntrackedParameter<unsigned int>("sampicSubDetId", 5)),
+      sampicSubDetId(conf.getParameter<unsigned int>("sampicSubDetId")),
       currentBlock(0),
       currentBlockValid(false) {
   for (const auto &it : conf.getParameter<vector<ParameterSet>>("configuration")) {

--- a/EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py
+++ b/EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py
@@ -8,6 +8,7 @@ totemTriggerRawToDigi.rawDataTag = cms.InputTag("rawDataCollector")
 totemDAQMappingESSourceXML_TrackingStrip = cms.ESSource("TotemDAQMappingESSourceXML",
   verbosity = cms.untracked.uint32(0),
   subSystem = cms.untracked.string("TrackingStrip"),
+  sampicSubDetId = cms.uint32(6),
   configuration = cms.VPSet(
     # 2016, before TS2
     cms.PSet(
@@ -62,6 +63,7 @@ totemRPRawToDigi.rawDataTag = cms.InputTag("rawDataCollector")
 totemDAQMappingESSourceXML_TimingDiamond = cms.ESSource("TotemDAQMappingESSourceXML",
   verbosity = cms.untracked.uint32(0),
   subSystem = cms.untracked.string("TimingDiamond"),
+  sampicSubDetId = cms.uint32(6),
   configuration = cms.VPSet(
     # 2016, before diamonds inserted in DAQ
     cms.PSet(
@@ -102,9 +104,9 @@ ctppsDiamondRawToDigi.rawDataTag = cms.InputTag("rawDataCollector")
 
 # ---------- Totem Timing ----------
 totemDAQMappingESSourceXML_TotemTiming = cms.ESSource("TotemDAQMappingESSourceXML",
-  verbosity = cms.untracked.uint32(10),
+  verbosity = cms.untracked.uint32(0),
   subSystem = cms.untracked.string("TotemTiming"),
-  sampicSubDetId = cms.untracked.uint32(5),
+  sampicSubDetId = cms.uint32(5),
   configuration = cms.VPSet(
     # 2017, before detector inserted in DAQ
     cms.PSet(
@@ -138,7 +140,7 @@ from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
 from Configuration.Eras.Modifier_ctpps_2017_cff import ctpps_2017
 from Configuration.Eras.Modifier_ctpps_2018_cff import ctpps_2018
 (ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(ctppsPixelDigis, isRun3 = False )
-(ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(totemDAQMappingESSourceXML_TotemTiming, sampicSubDetId = cms.untracked.uint32(6) )
+(ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(totemDAQMappingESSourceXML_TotemTiming, sampicSubDetId = 6)
 
 # raw-to-digi task and sequence
 ctppsRawToDigiTask = cms.Task(

